core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -1376,6 +1379,42 @@ public int getFileId(TachyonURI path) throws InvalidPathException {
     }
     return ret;
   }
+  
+  /**
+   * @return the total bytes on each storage tier.
+   */
+  public List<Long> getTotalBytesOnTiers() {
+    List<Long> ret = new ArrayList<Long>();
```

Comment:
Thanks, this is the normal way to instantiate ArrayList, the length of ArrayList cannot be instantiated. And, yes, We need to default the array list to zero from index 0 to # of storage level aliases everywhere because it will be computed later. And the length of array list equals the size of StorageLevelAlias.